ci/prow/make_config.go

```diff
@@ -906,24 +1063,33 @@ func indentBase(indentation int, prefix string, indentFirstLine bool, array []st
 		if i > 0 || indentFirstLine {
 			s += indent
 		}
-		s += prefix + quote(array[i]) + "\n"
+		if addQuote {
```

Comment:
Factor out the common part instead of duplicating the assignment.

Reply:
I have removed the flag. 
Now all the arrays in the generated yaml file will be quoted.